### DISCUSSION TITLE
Show always AxisTitles/SideTiltles

### DISCUSSION THREAD

**Discussion by TimoSchmuck** [imaNNeo/fl_chart#1188] [Ideas]
First of all this open source lib is great! Many thanks for this wonderfull pice of art and all the hard work going into it. 

What I'm missing (maybe I haven't figured it out) is the functionality to allways show the axisTitles, so that the graph can be scrolled horizontally, while keeping the axisTitles or SideTitles visible. This would increase readability of the chart, when not all data are visible and users have to scroll right.

**Comment by TimoSchmuck** [comment]
OK seems that this is a requested feature by many. According to the disscussion [https://github.com/imaNNeoFighT/fl_chart/issues/71] there is a private class to it, so forking could be a (bad) dirty solution.

[fl_chart/lib/src/chart/base/axis_chart/axis_chart_scaffold_widget.dart](https://github.com/imaNNeoFighT/fl_chart/blob/0d8ecade338f0a7dc501f60c18b2f18c0d63516d/lib/src/chart/base/axis_chart/axis_chart_scaffold_widget.dart#L7)

Line 7 in [0d8ecad](https://github.com/imaNNeoFighT/fl_chart/commit/0d8ecade338f0a7dc501f60c18b2f18c0d63516d)
 class AxisChartScaffoldWidget extends StatelessWidget {

**Comment by imaNNeo** [comment]
Yes, we made some fundamental changes to make us enable to implement the scroll feature by adding AxisChartScaffoldWidget in our private classes.
btw, we didn't implement it yet. 
Stay tuned and follow the #71
